src/cli.ts

```diff
@@ -144,14 +144,21 @@ program
   .description(
     'Push monitors to create new montors with Kibana monitor management UI'
   )
+  .option(
+    '--schedule <time>',
+    'schedule the monitors at configured intervals. Ex: setting `10m` configures the monitor to be executed every 10 minutes.'
```

Comment:
I was planning to keep this format similar to how Kibana expects instead of following HB syntax which is a bloated format anyway? Do you know the validation on the UI side?

Reply:
Lets move this to followup PR's if its non blocking to merge it with the bundler PR and keep it ready for E2E with Kibana.